src/java.base/share/classes/jdk/internal/reflect/ClassFileAssembler.java

```diff
@@ -49,7 +49,7 @@ public short getLength() {
     public void emitMagicAndVersion() {
         emitInt(0xCAFEBABE);
         emitShort((short) 0);
-        emitShort((short) 49);
+        emitShort((short) 62);
```

Comment:
This old bytecode generator does not support stack map and so these reflection classes would need to stay in version 49.   JEP 416 changes the core reflection implementation to build on top of method handles and no longer depends on this old bytecode generator.